eclass/kernel-build.eclass

```diff
@@ -337,6 +343,10 @@ kernel-build_src_install() {
 		targets+=( dtbs_install )
 	fi
 
+	if grep -q "CONFIG_CTF=y" "${WORKDIR}/modprep/.config"; then
+		targets+=( ctf_install )
```

Comment:
Should we also `ctf_install` in `src_test`?

Reply:
Good catch. It shouldn't matter much but it would be nice to ensure the target works correctly. I'll add that in.

Reply:
Done in 983fe336a7571f822655c25e0785f067de80c64c.